presto-main/src/main/java/com/facebook/presto/execution/SetPathTask.java

```diff
@@ -32,8 +43,32 @@ public String getName()
     }
 
     @Override
-    public ListenableFuture<?> execute(SetPath statement, TransactionManager transactionManager, Metadata metadata, AccessControl accessControl, QueryStateMachine stateMachine, List<Expression> parameters)
+    public ListenableFuture<?> execute(
+            SetPath statement,
+            TransactionManager transactionManager,
+            Metadata metadata,
+            AccessControl accessControl,
+            QueryStateMachine stateMachine,
+            List<Expression> parameters)
```

Comment:
please undo this change if there is nothing actually changed but formatting. It is nice to have formatting extracted as separate commits.